<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1093</width>
    <height>749</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1091</width>
      <height>171</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
     </font>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>30</y>
       <width>101</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    border: 1px solid #8f8f91;
    border-radius: 2px;
    background-color: rgb(66,133,244);
    min-width: 80px;
    color:#FFFFFF;
}

QPushButton:hover{
    border: 1px solid #8f8f91;
    border-radius: 3px;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,stop: 0 #5295F4, stop: 1 #4285F4);
    min-width: 80px;
}

QPushButton :pressed{
    border: 1px solid #8f8f91;
    border-radius: 2px;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,stop: 0 #5295F4, stop: 1 #4285F4);
    min-width: 80px;
    color:#FFFFFF;
}


</string>
     </property>
     <property name="text">
      <string>加载数据</string>
     </property>
    </widget>
    <widget class="QPushButton" name="BtnViewing">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>120</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    border: 1px solid #8f8f91;
    border-radius: 2px;
    background-color: rgb(234,67,53);
    min-width: 80px;
    color:#FFFFFF;
}

QPushButton:hover{
    border: 1px solid #8f8f91;
    border-radius: 3px;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,stop: 0 rgb(180,110,110), stop: 1 rgb(110,10,10));
    min-width: 80px;
}

QPushButton :pressed{
    border: 1px solid #8f8f91;
    border-radius: 2px;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,stop: 0 #4E6060, stop: 1 #1B1B1B);
    min-width: 80px;
    color:#FFFFFF;
}


</string>
     </property>
     <property name="text">
      <string>旋转</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinPointSize">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>40</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>20</number>
     </property>
     <property name="value">
      <number>9</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>点大小</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QCheckBox" name="ckbColored">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>颜色</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>90</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>观察角度</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>40</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>相机距离</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="spinDist">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>40</y>
       <width>70</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimum">
      <double>0.010000000000000</double>
     </property>
     <property name="maximum">
      <double>500.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.030000000000000</double>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="spinHor">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>90</y>
       <width>70</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-360.000000000000000</double>
     </property>
     <property name="maximum">
      <double>360.000000000000000</double>
     </property>
     <property name="value">
      <double>45.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="spinVer">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>90</y>
       <width>70</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-89.000000000000000</double>
     </property>
     <property name="maximum">
      <double>89.000000000000000</double>
     </property>
    </widget>
    <widget class="QCheckBox" name="ckbShowLine">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>显示KNN</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QComboBox" name="cbxHistories">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>90</y>
       <width>151</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>1101</width>
      <height>751</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>label_2</zorder>
   <zorder>groupBox</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
